### PR TITLE
Added #BlackLivesMatter banner.

### PR DESCRIPTION
**Pull request by carltongibson**
As per discussion on the DSF Members list, adds a banner expressing solidarity with the current protests in the USA to the main Django site and docs. 

Screenshot_2020-06-02 The Web framework for perfectionists with deadlines Django

It's perhaps not perfect but...

### CODE REVIEW

File: djangoproject/templates/base.html [hunk 1]
```diff
@@ -49,7 +49,22 @@
         {% block header %}{% endblock %}
       </div>
     </div>
-    <div id="billboard">{% block billboard %}{% endblock %}</div>
+    <div id="billboard">{% block billboard %}
+      <div style="background: #44B78B; color: #FFF; text-align:center;padding: 1em 3em;">
+        <p>
+          The Django Software Foundation deeply values the diversity of our developers, users, and community. We are distraught by the suyffering, oppression, and systemic racism the Black community faces every day. We can no longer remain silent.
```
Comment (evildmp) [inline djangoproject/templates/base.html:55]: _suffering_

Comment (carltongibson) [reply djangoproject/templates/base.html:55]: Thanks!

**Comment by carltongibson** [comment]
Screenshot_2020-06-02 The Web framework for perfectionists with deadlines Django(1)

I adjusted to use the dark green, after feedback on the mailing list.

**Comment by frankwiles** [comment]
LGTM!

**Comment by carltongibson** [comment]
OK, thanks @frankwiles.

**Pull request merged**